README.md

```diff
@@ -1,2 +1,316 @@
-# PSC2023-Capstone
- This repository is for an fictisious conference on my country. This is the final assigment of module 1 named Capstone Project.
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&display=swap');
+
+  /*Import box icons*/
+  @import url('https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css');
+
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+</style>
```

Comment:
- [ ] Since these codes are not playing important role in the README, I would suggest that you remove them 😃 this will make your README look good 👍